core/src/main/java/nu/yona/server/util/TimeUtil.java

```diff
@@ -1,12 +1,10 @@
 /*******************************************************************************
- * Copyright (c) 2017 Stichting Yona Foundation
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ * Copyright (c) 2017 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
```

Comment:
I agree, but that's what automatically happens when the code formatter runs. The copyright inserter formats it nicely to start with but than the formatter reformats it. Legally nothing changes, so I think it's fine.